docs/source/configuration.rst

```diff
@@ -22,13 +22,7 @@ file processed takes precedence. List-valued directives (such as
 Configuration Discovery
 -----------------------
 
-``tartufo`` uses two additional heuristics to locate specified configuration files.
-First, the current working directory is used as a base for relative filenames.
```

Comment:
Was this logic removed? It looks to me like only the parent traversal logic was removed, not the logic that tries in the current working directory.

Reply:
It's not trying in the current working directory per se. It's saying that pathnames provided with `--config` are interpreted relative to the current working directory.

Reply:
Looking at the entirety of the deleted text, it's probably worth adding an explicit statement to that effect (although I suppose it should be obvious). i.e. "Path names specified using `--config` are interpreted relative to the current working directory."

Reply:
Apologies, I think I just misread what the original text here was conveying. Looking one more time at the documentation, I don't have any objections at all to this removal.

The examples in https://github.com/sushantmimani/tartufo/blob/remove-upward-traversal/docs/source/configuration.rst#configuration-discovery do need to be updated though, right? They speak at least twice about traversing upwards still.

Reply:
Documentation has been updated.